src/lib/poracleMessage/commands/disable.js

```diff
@@ -22,7 +22,7 @@ exports.run = async (client, msg, args, options) => {
 		for (const id of targets) {
 			client.log.info(`Disable ${id}`)
 
-			await client.query.updateQuery('humans', { admin_disable: 1, disabled_date: null }, { id })
+			await client.query.updateQuery('humans', { admin_disable: 1, disabled_date: null }, { id: id })
```

Comment:
Doesn't this just get changed back by lint? It certainly would by webstorm. It's equivalent to what was there before

Reply:
You're right. This was test code to attempt to isolate what was going on. This has been reverted.